Externals/glslang/glslang/MachineIndependent/ShaderLang.cpp

```diff
@@ -1268,21 +1267,9 @@ const char* ShGetInfoLog(const ShHandle handle)
 // Return the resulting binary code from the link process.  Structure
 // is machine dependent.
 //
-const void* ShGetExecutable(const ShHandle handle)
```

Comment:
this change is kind of scary, but every path just seems to lead to returning nullptr...